src/styles/popover.scss

```diff
@@ -236,14 +244,20 @@ $block: #{$fd-namespace}-popover;
     @include fd-reset();
 
     border: none;
-    position: absolute;
+    position: relative;
+    width: 100%;
     white-space: nowrap;
     box-shadow: $fd-popover-box-shadow;
     z-index: $fd-popover-z-index;
     border-radius: $fd-popover-border-radius;
     background: $fd-popover-background-color;
     opacity: 1;
 
+    &.fd-inline-help__content {
```

Comment:
this is not allowed in styles. It's breaking the self-contained rule, which means that you can't modify the properties of one component inside another component. In this case you can't modify Inline Help content inside the block code for Popover.